src/components/common/Extensions/ImageCarousel/ImageCarousel.tsx

```diff
@@ -42,13 +57,23 @@ const ImageCarousel: FC<ImageCarouselProps> = ({
             // eslint-disable-next-line react/no-array-index-key
             key={index}
             onClick={() => scrollTo(index)}
-            className={clsx(
-              'mx-1 h-2 w-2 cursor-pointer rounded-full bg-gray-200 transition-all duration-normal hover:bg-blue-400',
-              {
-                'bg-gray-500': index === selectedIndex,
-              },
-            )}
-          />
+            className={clsx('group', {
+              'py-[10px]': !isChangeSlideDotButton,
```

Comment:
could we just render a whole different thing if `isChangeSlideDotButton` is false?